src/titiler/core/titiler/core/middleware.py

```diff
@@ -31,6 +32,7 @@ def __init__(
         """
         super().__init__(app)
         self.cachecontrol = cachecontrol
+        self.cachecontrol_http_code_range = cachecontrol_http_code_range
```

Comment:
I'm not sure about the name of the parameter. To me if we say `Range` I'm assuming the input to be a sequence (min, max), while here in reality we want the upper value of status_code.

what do you think about `cachecontrol_max_http_code` ?

Reply:
Strong agree 👍 .